Makefile

```diff
@@ -1,3 +1,5 @@
+-include Makefile.package-stanc
```

Comment:
Is there a reason why the packaging makefile couldn't live entirely inside the bytecode build directory? It seems unnecessary to include any of this in the top-level makefile since it won't be the default way of building stanc